src/js/control-bar/text-track-controls/text-track-menu-item.js

```diff
@@ -34,13 +33,14 @@ class TextTrackMenuItem extends MenuItem {
     super(player, options);
 
     this.track = track;
-    const changeHandler = Fn.bind(this, this.handleTracksChange);
-    const selectedLanguageChangeHandler = Fn.bind(this, this.handleSelectedLanguageChange);
+    const changeHandler = (...args) => this.handleTracksChange.apply(this, args);
```

Comment:
these two methods were binding the prototype methods so if any TextTrackMenuItem was disposed, all the event handlers were removed.